common/djangoapps/course_modes/__init__.py

```diff
@@ -0,0 +1,4 @@
+"""
+Setup the signals on startup.
+"""
+import course_modes.signals
```

Comment:
It might be better to do this in the `startup.py` module, rather than in `__init__.py`, so that you're relying less on side effects of importing.